buildSrc/src/main/java/Dependencies.kt

```diff
@@ -29,7 +29,7 @@ private object Versions {
     const val androidx_transition = "1.1.0-rc01"
     const val google_material = "1.1.0-alpha06"
 
-    const val mozilla_android_components = "0.54.0-SNAPSHOT"
+    const val mozilla_android_components = "0.55.0-SNAPSHOT"
```

Comment:
`mozilla_appservices` needs to be updated to '0.29.0' as well.

Reply:
Yup, it's quite unfortunate we have a-s version in fenix again, even if just for tests :/

Reply:
And fix some breaking changes related to `AccountObserver` https://mozac.org/changelog/#0550


> ⚠️ This is a breaking behavior change: API changes to facilitate error handling; new method on AccountObserver interface.

Reply:
We noticed this in some work we were doing async as well and thought I should let you know. :)